systemd/kano-common.target

```diff
@@ -12,6 +12,6 @@
 Description=Kano Common User Services
 IgnoreOnIsolate=true
 Wants=kano-common-tracker.service kano-common-notifications.service \
- kano-common-speakerleds.service kano-common-boards.service \
+ kano-boards.service kano-speakerleds.service \
```

Comment:
We can reverse the dependency for clarity by using 

```
WantedBy=kano-common.target
```

in [kano-boards.service](https://github.com/KanoComputing/kano-peripherals/blob/master/systemd/user/kano-boards.service#L14) and [kano-speakerleds.service](https://github.com/KanoComputing/kano-peripherals/blob/master/systemd/user/kano-speakerleds.service#L17) such that `kano-desktop` won't need to know anything about `kano-peripherals`.